Mathlib/RepresentationTheory/Basic.lean

```diff
@@ -272,6 +272,37 @@ instance : IsScalarTower k (MonoidAlgebra k G) ρ.asModule where
 
 end MonoidAlgebra
 
+section Norm
+
+variable {k G V : Type*} [CommSemiring k] [Group G] [Fintype G] [AddCommMonoid V] [Module k V]
+variable (ρ : Representation k G V)
+
+/-- Given a representation `(V, ρ)` of a finite group `G`, this is the linear map `V →ₗ[k] V`
+defined by `x ↦ ∑ ρ g x` for `g` in `G`. -/
+def norm : V →ₗ[k] V := ∑ g : G, ρ g
+
+@[simp]
+lemma norm_comp_self (g : G) :
+    norm ρ ∘ₗ ρ g = norm ρ := by
+  ext
+  simpa [norm] using Fintype.sum_bijective (· * g) (Group.mulRight_bijective g) _ _ <| by simp
+
+@[simp]
+lemma norm_self_apply (g : G) (x : V) :
+    norm ρ (ρ g x) = norm ρ x := LinearMap.ext_iff.1 (norm_comp_self _ _) x
```

Comment:
```suggestion
lemma norm_self_apply (g : G) (x : V) : norm ρ (ρ g x) = norm ρ x :=
  LinearMap.ext_iff.1 (norm_comp_self _ _) x
```
and similarly the next two lemmas. Isn't that the library style?

Reply:
done, thanks